### DISCUSSION TITLE
add suport for the rp2040 zero from waveshare

### DISCUSSION THREAD

**Discussion by pascal2313** [tttapa/Control-Surface#1013] [Q&A]
add support for the rp2040 zero form waveshare becuse it is an tiny and realy cheap board you can fit it in every thing it also has just as many pins as the raspberry pico

**Comment by tttapa** [comment]
I don't have such a board to test anything, but the Control Surface examples seem to compile without issues using https://github.com/earlephilhower/arduino-pico.

**Reply by pascal2313** [reply]
it compiles with out issue but if i programm it it does not work as an midi device. and if i program it using the raspberry pi pico core it is working but the pins are all wrong

**Reply by tttapa** [reply]
Please post a link to the core that works. Also enable verbose output during compilation in the IDE preferences, and post the FQBN that is printed at the beginning of the output when you verify or upload the sketch (for both of the scenarios you described).

**Reply by pascal2313** [reply]
this is the core that is working (Arduino Mbed Core for RP2040 boards)

https://github.com/earlephilhower/arduino-pico this is the core with the rp2040 zero

the working FQBN
FQBN: arduino:mbed_rp2040:pico
Using board 'pico' from platform in folder: C:\Users\pascal_pc\AppData\Local\Arduino15\packages\arduino\hardware\mbed_rp2040\4.1.1
Using core 'arduino' from platform in folder: C:\Users\pascal_pc\AppData\Local\Arduino15\packages\arduino\hardware\mbed_rp2040\4.1.1

the non working FQBN
FQBN: rp2040:rp2040:waveshare_rp2040_zero
Using board 'waveshare_rp2040_zero' from platform in folder: C:\Users\pascal_pc\AppData\Local\Arduino15\packages\rp2040\hardware\rp2040\3.7.2
Using core 'rp2040' from platform in folder: C:\Users\pascal_pc\AppData\Local\Arduino15\packages\rp2040\hardware\rp2040\3.7.2

sorry for the late reply
